src/main/features/ccj/form/models/reDetermination.ts

```diff
@@ -0,0 +1,30 @@
+import { IsDefined } from 'class-validator'
+
+export class ValidationErrors {
+  static readonly TYPE_REQUIRED: string = 'Enter a valid reason'
+}
+
+export class ReDetermination {
+  @IsDefined({ message: ValidationErrors.TYPE_REQUIRED })
```

Comment:
It's not type, please rename `TYPE_REQUIRED` to `REASON_REQUIRED` or `EXPLANATION_REQUIRED` etc

Reply:
done